lib/uefi/boot_service_provider.h

```diff
@@ -150,6 +150,10 @@ struct EFI_LOADED_IMAGE_PROTOCOL {
 };
 
 static constexpr size_t EFI_LOADED_IMAGE_PROTOCOL_REVISION = 0x1000;
+static constexpr size_t EFI_DEVICE_PATH_TYPE_END = 0x7f;
```

Comment:
These are only used in debug_support.cpp, can we move them to debug_support.cpp?